index.html

```diff
@@ -4928,13 +4928,21 @@ <h3>Fitness for Purpose</h3>
     <section class="appendix informative">
       <h2>Contexts, Types, and Credential Schemas</h2>
 
-      <section class="informative">
+      <section class="normative">
         <h3>Base Context</h3>
+
+        <p class="issue" title="(AT RISK) Hash values might change during Candidate Recommendation">
+This section lists cryptographic hash values that might change during the
+Candidate Recommendation phase based on implementer feedback that requires
+the referenced files to be modified.
+        </p>
           <p>
-The base context, located at <code>https://www.w3.org/ns/credentials/v2</code>
-with a SHA-256 digest of
-<strong><code>ff1fdfa8d3f07547ec149b6652fe3f5a4276b9f0c48344dc29400763ba7b44fa</code></strong>,
-can be used to implement a local cached copy. It is possible to confirm the
+Implementations MUST ensure that the base context value, located at
+<code>https://www.w3.org/ns/credentials/v2</code>, matches the following
+SHA-256 digest of the value:
+<strong><code>dc28cf86d8a3199beb9d14e29d1da662346b8660abf2edca873098c88ca2a457</code></strong>.
```

Comment:
Use the same approach for data integrity and context integrity see https://github.com/w3c/vc-data-model/pull/1140

Reply:
https://github.com/w3c/vc-data-model/blob/main/contexts/credentials/v2#L199

Reply:
> Rational for "why" alignment matters.
> 
> Implementers are required to load dependencies to support normative statements, when implementing the specification.
> 
> It should be a design goal of this working group to reduce the amount of vendor specific libraries (non standard implementations) that an implementation must use to implement this standard.
> 
> If implementers are already required to load a library that supports multibase (even a trivial library implemented by a startup), then they should be able to use that library to implement support for normative requirements the same way, everywhere they show up.
> 
> Here is a counter example (showing what not to do):
> 
> * v2 core context secured with sha512 hex encoded.
> * resourceIntegrity feature secured with sha384 (base64url encoded)
> * proofValue in data integrity secured with multibase (base58btc encoded)

i agree in principle and think that there is a desire to have this pr (or a follow on to it) to align with #1140 and vice versa.  This would mean that our primary normative doc (the spec itself) is consistent.


regarding the reference to proof value (i believe only a reference from core data perspective in the context) i think that this should be handled separately (e.g. what goes in the context since we agreed it is normative?) 

Broadly, i think the questions of encodings for data integrity and what is done under the data integrity spec belong there.

Reply:
> (e.g. what goes in the context since we agreed it is normative?)

I agree, but unfortunately, we are *making the context normative with logs of things already in it*... so your solution does not work.

> Broadly, i think the questions of encodings for data integrity and what is done under the data integrity spec belong there.

I also agree with this, but the working group disagreed, when I attempted to separate things.

So now we have to make them consistent, or someone else needs to try to separate them in a way so that they can diverge without causing further implementer burden.

Reply:
@OR13 — `s/logs of things already in it/lots of things already in it/` (that is, `logs`->`lots`)?